src/arg_desc.jl

```diff
@@ -1,187 +1,187 @@
-:xerror                      => "AbstractVector or 2-Tuple of Vectors. x (horizontal) error relative to x-value.  If 2-tuple of vectors, the first vector corresponds to the left error (and the second to the right)",
-:yerror                      => "AbstractVector or 2-Tuple of Vectors. y (vertical) error relative to y-value.  If 2-tuple of vectors, the first vector corresponds to the bottom error (and the second to the top)",
-:ribbon                      => "Number or AbstractVector.  Creates a fillrange around the data points.",
-:quiver                      => "AbstractVector or 2-Tuple of vectors.  The directional vectors U,V which specify velocity/gradient vectors for a quiver plot.",
-:arrow                       => "nothing (no arrows), Bool (if true, default arrows), Arrow object, or arg(s) that could be style or head length/widths.  Defines arrowheads that should be displayed at the end of path line segments (just before a NaN and the last non-NaN point).  Used in quiverplot, streamplot, or similar.",
-:normalize                   => "Bool or Symbol. Histogram normalization mode. Possible values are: false/:none (no normalization, default), true/:pdf (normalize to a discrete Probability Density Function, where the total area of the bins is 1), :probability (bin heights sum to 1) and :density (the area of each bin, rather than the height, is equal to the counts - useful for uneven bin sizes).",
-:weights                     => "AbstractVector. Used in histogram types for weighted counts.",
-:show_empty_bins             => "Bool. Whether empty bins in a 2D histogram are colored as 0 (true), or transparent (the default)",
-:contours                    => "Bool. Add contours to the side-grids of 3D plots?  Used in surface/wireframe.",
-:contour_labels              => "Bool. Show labels at the contour lines?",
-:match_dimensions            => "Bool. For heatmap types... should the first dimension of a matrix (rows) correspond to the first dimension of the plot (x-axis)?  The default is false, which matches the behavior of Matplotlib, Plotly, and others.  Note: when passing a function for z, the function should still map `(x,y) -> z`.",
-:subplot                     => "Integer (subplot index) or Subplot object.  The subplot that this series belongs to.",
-:series_annotations          => "AbstractVector of String or PlotText.  These are annotations which are mapped to data points/positions.",
-:primary                     => "Bool.  Does this count as a 'real series'?  For example, you could have a path (primary), and a scatter (secondary) as 2 separate series, maybe with different data (see sticks recipe for an example).  The secondary series will get the same color, etc as the primary.",
-:hover                       => "nothing or vector of strings. Text to display when hovering over each data point.",
-:colorbar_entry              => "Bool.  Include this series in the color bar?  Set to `false` to exclude.",
```

Comment:
I think we can skip formatting for this whole file also

Reply:
better use `align_pair_arrow = true`